blocks/core/_shared/settings.less

```diff
@@ -19,6 +19,7 @@ and shared between all themes */
 
 @ff_colour_control_bg: @ff_colour_quiet;
 @ff_colour_control_bg_selected: darken(@ff_colour_quiet, 10%);
+@ff_colour_control_bg_lighter: lighten(@ff_colour_quiet,5%);
```

Comment:
hmm. Not sure about this. @dkhuntrods what do you think? Make a new variable here, or in the module?